temp_streaming_function.py

```
@@ -0,0 +1,394 @@
+            
+            user_progress.orchestrator_data['state'] = state_dict
+            from sqlalchemy.orm.attributes import flag_modified
+            flag_modified(user_progress, "orchestrator_data")
+            user_progress.last_activity = datetime.utcnow()
+            db_session.commit()
+            print(f"[STREAM DEBUG] Committed to database. Turn count: {orchestrator.state.turn_count}, simulation_started: {orchestrator.state.simulation_started}")
+            
+            # Send final metadata
+            yield f"data: {json.dumps({'done': True, 'persona_name': persona_name, 'persona_id': str(persona_id) if persona_id else None, 'scene_completed': scene_completed, 'next_scene_id': next_scene_id, 'turn_count': orchestrator.state.turn_count, 'full_content': full_response})}\n\n"
+            
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)